serverless-http.js

```diff
@@ -16,6 +16,10 @@ module.exports = function(app, opts) {
   const options = Object.assign({}, defaultOptions, opts);
 
   return (evt, ctx, callback) => {
+    if (options.callbackWaitsForEmptyEventLoop != null) {
```

Comment:
I choose to keep it simpler, because this is how the aws-serverless-express library does it, and knowing what things will create events in the loop can be tricky (and change from package to package) and manifests as your functions hanging sometimes.